.github/workflows/publish_deploy_image.yaml

```diff
@@ -1,11 +1,16 @@
-name: Create and publish a Docker image to ghrc.io
+name: Create and publish a Docker image to ghrc.io and deploy to openshift
 
 on:
-  push
+  push:
+    branches:
+    - main
```

Comment:
This looks good for now, since it's exclusively going to `dev`, and let's make sure we have a ticket for [adding support for approvals](https://docs.github.com/en/actions/managing-workflow-runs/reviewing-deployments) as we look at `test` and `prod` deployment.